apps/frontend/src/app/booking-confirmation/[bookingId]/page.tsx

```diff
@@ -15,29 +15,39 @@ async function ConfirmationPage(
   props: CurrentUserProps & { params: Promise<{ bookingId: string }> },
 ) {
   return (
-    <div className="flex min-h-dvh flex-col">
-      <div className="relative flex h-32 items-center justify-center overflow-hidden">
-        <div className="-z-10 absolute h-full w-[120%] rounded-b-[50%] bg-secondary md:w-[110%]" />
+    <VStack minHeight="100dvh">
+      <Center height={32} position="relative">
+        <Box
+          backgroundColor="gray.50"
+          height="full"
+          position="absolute"
+          roundedBottom="50%"
+          width={{ base: "120%", md: "110%" }}
+          zIndex={-10}
+        />
+
         <Link href="/">
           <UabcHeaderText className="mt-4" />
         </Link>
-      </div>
+      </Center>
 
-      <div className="mx-4 flex grow flex-col items-center justify-center gap-6 py-10">
+      <VStack alignItems="center" flexGrow={1} justifyContent="center" paddingY={10}>
         {
           /**
            * // TODO
            */
           // @ts-ignore
           <ConfirmationMessage email={await props.currentUser.email} member={false} />
         }
-        <Link className={buttonVariants({ variant: "ghost" })} href="/sessions">
-          Return Home
-        </Link>
-      </div>
+        <Button variant="ghost">
+          <Link href="/sessions">Return Home</Link>
+        </Button>
+      </VStack>
 
-      <div className="flex flex-col gap-4 bg-primary p-4 pt-5 pb-6">
-        <p className="font-semibold text-primary-foreground text-xl">Rackets at the ready!</p>
+      <VStack backgroundColor="primary" padding="md">
```

Comment:
If we're using `Container` at top, we can also wrap this VStack with Bleed.
```suggestion
      <Bleed block="md" inline="md">
        <VStack backgroundColor="primary" padding="md">
```